File: 프로그래머스/0/181862. 세 개의 구분자/세 개의 구분자.cs
using System;

public class Solution {
    public string[] solution(string myStr) {
        myStr = myStr.Replace("a"," ").Replace("b", " ").Replace("c"," ");
        string[] answer = myStr.Split(" ",System.StringSplitOptions.RemoveEmptyEntries);
         if(answer.Length == 0)
         {
             answer = new string[1];
             answer[0] = "EMPTY";
         }
            
        return answer;
    }
}

File: 프로그래머스/0/181862. 세 개의 구분자/세 개의 구분자.cpp
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>
using namespace std;

vector<string> solution(string myStr) {
    vector<string> answer;
    replace(myStr.begin(),myStr.end(),'a',' ');
    replace(myStr.begin(),myStr.end(),'b',' ');
    replace(myStr.begin(),myStr.end(),'c',' ');
    stringstream ss(myStr);
    string temp;
    while(getline(ss,temp,' '))
    {
        if(!temp.empty())
            answer.push_back(temp);
    }
    if(answer.size() == 0)
        answer.push_back("EMPTY");
    return answer;
}